packages/react-router/modules/Match.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import invariant from "invariant";
+
+import RouterContext from "./RouterContext";
+import Route, { getContext } from "./Route";
+
+class Match extends React.Component {
+  render() {
+    Route.displayName = "Match";
```

Comment:
Changing `Route.displayName` here is going to change it everywhere. Can you explain a bit about why you did this?